script.js

```diff
@@ -318,3 +319,37 @@ form.addEventListener("submit", (e) => {
       "Please, use only lowercase letter for your email!";
   }
 });
+
+window.addEventListener('load', () => {
+  if (localStorage.getItem('userInfo')) {
+    const userInfo = JSON.parse(localStorage.getItem('userInfo'));
+    usernameInput.value = userInfo.name;
+    emailInput.value = userInfo.email;
+    messageText.value = userInfo.message;
+  } else {
+    localStorage.setItem('userInfo', userData);
+  }
+});
```

Comment:
* This error appears in the console when the input fields are populated 
* Please fix 🙏 
![Screenshot (120)](https://user-images.githubusercontent.com/52497246/124192088-a14fcb00-dabc-11eb-9f14-716c8c5368d9.png)

Reply:
_fixed the errors_